lang/python/python-pyparsing/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (C) 2019 CZ.NIC z.s.p.o. (http://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-pyparsing
+PKG_VERSION:=2.4.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=pyparsing
+PKG_HASH:=6f98a7b9397e206d78cc01df10131398f1c8b8510a2f4d97d9abd82e1aacdd80
+PKG_BUILD_DIR:=$(BUILD_DIR)/pyparsing-$(PKG_VERSION)
```

Comment:
Setting `PKG_BUILD_DIR` shouldn't be necessary when using `pypi.mk`.

Reply:
Removed